eng/Publishing.props

```diff
@@ -27,9 +27,10 @@
         Condition=" '$(PublishInstallerBaseVersion)' == 'true' " />
     <_InstallersToPublish Include="$(ArtifactsDir)\installers\**\*.wixlib" UploadPathSegment="Runtime" />
     <_InstallersToPublish Include="$(ArtifactsDir)\installers\**\*.zip" UploadPathSegment="Runtime" />
+    <_ChecksumsToPublish Include="$(ArtifactsDir)\**\*$(ChecksumExtension)" />
```

Comment:
Can't see how use of `$(ChecksumExtension)` will work unless Arcade defines it in the post-built. See https://github.com/dotnet/aspnetcore/blob/1897a5d816695fdb10a918e80236507b38de5186/eng/Publishing.props#L11

Reply:
Whoops, didn't realize that was the case. I added that commit as a cleanup item after I got a good internal build - will fix